src/azul/project/hca/transformers.py

```diff
@@ -106,6 +106,7 @@ def _file_dict(file: api.File) -> JSON:
 
 def _protocol_dict(protocol: api.Protocol) -> JSON:
     protocol_dict = {"document_id": protocol.document_id}
+    protocol_dict['protocol_id'] = protocol.protocol_id
```

Comment:
I have the feeling we should only include the `protocol_id` from `AnalyisProtocol` instances. 

If we don't do this, we'll get all those garbage-y `library_preparation_protocol_1` that would clutter up the facet drop down in the browser.

We may also want to introduce two keys by splitting `AnalyisProtocol.protocol_id` at `'_'` and assigning to `protocol_dict['workflow_name']` and `protocol_dict['workflow_version']`. Not sure about this at all. It would introduce another "nested facet". cc:@NoopDog

Reply:
Modified so that only AnalysisProtocol objects will have their `protocol_id` value split up on the last underscore and stored in `protocol_dict['workflow']` and `protocol_dict['workflow_version']`.